apps/rule-manager/app/db/DbRule.scala

```diff
@@ -140,7 +141,7 @@ object DbRule extends SQLSyntaxSupport[DbRule] {
       description: Option[String] = None,
       ignore: Boolean,
       notes: Option[String] = None,
-      googleSheetId: Option[String] = None,
+      externalId: String = UUID.randomUUID().toString(),
```

Comment:
Would it be better to keep this as `Option[String]` and let the database generate the default value?